### PR TITLE
Add Korean Translation

### PR DESCRIPTION
**Pull request by green1052**
_If your pull request is a translation update, then the title of this pull must contains 'string' or 'translation'_

### CODE REVIEW

**Review by MlgmXyysd — CHANGES_REQUESTED** [review]
Only keep ko/strings.xml, delete others

**Comment by green1052** [comment]
I did.

File: app/src/main/res/values-ko/strings.xml [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+    <string name="module_activation_scope">Activation Scope</string>
+    <string name="enable_scope">Enable for this module</string>
+    <string name="scope_notice">To enable the activation scope of this module, turn Activation Scope on and select applications.</string>
+    <string name="item_list_filter">Filter</string>
+    <string name="menu_show_no_name">Show apps without names</string>
+    <string name="menu_show_no_icon">Show apps without icon</string>
+    <string name="menu_show_user_apps">Show user apps</string>
+    <string name="settings_theme_dark_auto">Follow system</string>
+    <string name="settings_summary_pure_black">Use the pure black theme if dark theme is enabled</string>
+    <string name="settings_title_use_sandhook">Use SandHook variant backend</string>
```
Comment (MlgmXyysd) [inline app/src/main/res/values-ko/strings.xml:421]: Incomplete translation

File: app/src/main/res/values-ko/strings.xml [hunk 2]
```diff
@@ -0,0 +1,423 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
+
+    <!-- Do NOT translate these -->
+    <string name="app_name" translatable="false">EdXposed Manager</string>
```
Comment (MlgmXyysd) [inline app/src/main/res/values-ko/strings.xml:5]: Remove tags which with 'translatable="false"'

**Review by MlgmXyysd — CHANGES_REQUESTED** [review]

**Pull request merged**